hieradata/site/cp/role/htcondor.yaml

```diff
@@ -3,11 +3,31 @@ htcondor::htcondor_host: "htcondor-cm.cp.lsst.org"
 
 nfs::client_enabled: true
 nfs::client_mounts:
-  /mnt/project:
+  /project:
```

Comment:
Why is the mount being changed to be `/project`?

Reply:
everyone complained that the environment needs to be the same as other instances they have for the software stack... there is a huge discussion on this on the htcondor channel, it's part of the things I asked people to test on dev but nobody gave feedback to do it while this was been developed.